### PR TITLE
qbs: port build to CMake

### PR DESCRIPTION
**Pull request by ABBAPOH**
- [+] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [+] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [+] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [+] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [+] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [+] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/qbs.rb [hunk 1]
```diff
@@ -4,6 +4,7 @@ class Qbs < Formula
   url "https://download.qt.io/official_releases/qbs/1.20.0/qbs-src-1.20.0.tar.gz"
   sha256 "44961a4bb61580ae821aaf25ebb5a5737bd8fb79ec0474aa2592cdd45cc5171f"
   license :cannot_represent
+  revision 1
```
Comment (carlocab) [inline Formula/qbs.rb:7]: Does using CMake have any user-visible changes? If it doesn't, we can omit the `revision`.

Comment (ABBAPOH) [reply Formula/qbs.rb]: done, thanks

**Comment by carlocab** [comment]
Thanks. One more question: what's the advantage to using `cmake` instead of `qmake`? Is the `qmake` system being replaced for QBS?

**Comment by ABBAPOH** [comment]
Yes, this is in-line with the transition of the Qt project to CMake (and QtCreator which also ships Qbs). Besides, it's hard to maintain 3 buildsystems (Qbs also can be build with Qbs along with QMake and CMake), so I hope we can retire qmake one day

**Review by carlocab — APPROVED** [review]
> Besides, it's hard to maintain 3 buildsystems (Qbs also can be build with Qbs along with QMake and CMake), so I hope we can retire qmake one day

Sounds good. Happy to help you put a nail in the coffin. Thanks for keeping the formula updated!

**Pull request closed without merging**